srv/adapter/kobold.ts

```diff
@@ -25,6 +25,10 @@ export const handleKobold: ModelAdapter = async function* ({
 }) {
   const body = { ...base, ...settings, prompt }
 
+  // Kobold has a stop requence parameter which automatically
+  // halts generation when a certain token is generated
+  body.stop_sequence = getEndTokens(char, members, ['END_OF_DIALOG','You:'])
```

Comment:
What happens if a stop token occurs at the beginning for a response?
E.g. the stop tokens likely includes `Botname:`, but the response begins with `Botname:` would the response immediately stop?

Reply:
It would, yes.

Reply:
I don't think the `replyAs.name` needs to be a stop sequence. Every other character name + member names should do the trick.